src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/methodparampad/InputMethodParamPad.java

```diff
@@ -79,5 +79,5 @@ public void newArray()
             ()
         {
         }
-    }
+    }	
```

Comment:
Oh.It's a truly new level of development for me.Everything metters even spaces that visually and practically doesn't make a difference actually is important.I should get used to...I spent for task that could be done in 5-10 minutes few days.. xD